src/main/java/seedu/address/model/util/SampleDataUtil.java

```diff
@@ -6,12 +6,7 @@
 
 import seedu.address.model.AddressBook;
 import seedu.address.model.ReadOnlyAddressBook;
-import seedu.address.model.person.Address;
-import seedu.address.model.person.Email;
-import seedu.address.model.person.Flag;
-import seedu.address.model.person.Name;
-import seedu.address.model.person.Person;
-import seedu.address.model.person.Phone;
+import seedu.address.model.person.*;
```

Comment:
It would be better to not optimise the imports.

Reply:
Ok fixed